static/js/pm_list.js

```diff
@@ -59,52 +78,157 @@ export function update_private_messages() {
 
         vdom.update(replace_content, find, new_dom, prior_dom);
         prior_dom = new_dom;
+    } else {
+        // When we update the private_messages_section in its collapsed state,
+        // the only thing to check is if there's an active conversation thread to
+        // display outside the collapsed section.
+        update_collapsed_private_messages_section();
+
+        // Also, change the toggle icon of private_messages_section to collapsed one, since
+        // we have `private_messages_collapsed` as `true`.
+        $("#toggle-private-messages-section-icon").removeClass("fa-caret-down");
+        $("#toggle-private-messages-section-icon").addClass("fa-caret-right");
```

Comment:
These addClass/removeClass actions should happen when we update `private_messages_collapsed`, since that will ensure we do them exactly once when changing that state.

Reply:
@timabbott isn't this is what written ?
the code would come in this `else` condition only if `private_messages_collapsed` is true;

Reply:
@timabbott , isn't this is what written here?
The code would come in this `else` condition only when we have `private_messages_collapsed ` as true;
Please correct me If I'm wrong somewhere;

Reply:
See the commit to squash I've pushed for what I have in mind.

Reply:
Cool; Thanks!